config/locales/fil/signs.fil.yml

```diff
@@ -0,0 +1,8 @@
+fil:
+  pages:
+    signs:
+      title: 'Nada-download na Mga Palatandaan ng Banyo'
+      intro: 'Kung mayroon kang pampublikong banyo sa iyong gusali o negosyo na neutral sa kasarian, ipaalam sa mga tao. Ikaw (at / o ang may-ari / pamamahala ng gusali) ay maaaring idagdag ito sa database, at mag-post ng isa sa mga tanda sa ibaba upang ipaalam sa mga tao na sinusuportahan mo ang ligtas na pag-access para sa lahat. Ang parehong mga palatandaan ay maaaring i-print na mga PDF file.'
```

Comment:
> print

Not sure if something else, like "ilimbag" or "imprentahin," could be better than "print"?